crates/flux-middle/src/fhir.rs

```diff
@@ -230,7 +230,7 @@ impl<'fhir> Item<'fhir> {
 pub enum ItemKind<'fhir> {
     Enum(EnumDef<'fhir>),
     Struct(StructDef<'fhir>),
-    TyAlias(TyAlias<'fhir>),
+    TyAlias(Box<TyAlias<'fhir>>),
```

Comment:
You should allocate this in the arena

```suggestion
    TyAlias(&'fhir TyAlias<'fhir>),
```